mithril-ui/menu/Inventory.js

```diff
@@ -47,14 +47,27 @@ const InspectPanel = (initialVnode) => {
 };
 
 const ActionSlots = (initialVnode) => {
+  const onSlotClick = (slot) => {
+    actionSlotsActions.setActionSlot({
+      id: slot.id,
+      image: !slot.image ? menuState.selectedItem.preview_url : null
```

Comment:
I don't know if it makes sense as a toggle in the final version but this works for testing.

Reply:
agreed, mostly for testing